src/components/Favorites/index.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import {
+    importSavedFavorites,
+    removeAllExistingFavorites,
+    removeExistingFavorite,
+} from "../../features/favoritesSlice";
+import styles from "./favorites.module.css";
+import { useDispatch, useSelector } from "react-redux";
+import { AiOutlineStar } from "react-icons/ai";
+import { fetchSingleCoin } from "../../features/singleCoinSlice";
+
+function Favorites() {
+    const { data, fetched } = useSelector((state) => state.favorites);
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        if (!data?.length && !fetched) {
+            dispatch(importSavedFavorites());
+        }
+    }, [dispatch, data, fetched]);
+
+    const removeFavorite = (coin) => {
+        dispatch(removeExistingFavorite(coin));
+    };
+    const clearFavorites = () => {
+        dispatch(removeAllExistingFavorites());
+        dispatch(importSavedFavorites());
+    };
+    const coinFetchFromFavs = (coin) => {
+        dispatch(fetchSingleCoin(coin));
+    };
+
+    if (data?.length) {
+        return (
+            <section className={styles.sidePanel}>
+                <h2>Favorites</h2>
+                <ul>
+                    {data.map((coin, index) => (
+                        <li key={index}>
+                            {coin}
```

Comment:
dont leave floating text unwrapped. `span` or `p` tags

Reply:
this is being addressed in styling branch.